From: Colin Ian King <colin.king@canonical.com>
Date: Sat, 28 Apr 2018 10:52:16 +0100
Subject: liquidio: fix spelling mistake: "mac_tx_multi_collison" ->
 "mac_tx_multi_collision"
Patch-mainline: v4.18-rc1
Git-commit: 76c2a96d42ca3bdac12c463ff27fec3bb2982e3f
References: bsc#1119406 FATE#326562

Trivial fix to spelling mistake in oct_stats_strings text

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/cavium/liquidio/lio_ethtool.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/cavium/liquidio/lio_ethtool.c
+++ b/drivers/net/ethernet/cavium/liquidio/lio_ethtool.c
@@ -122,7 +122,7 @@ static const char oct_stats_strings[][ET
 	"mac_tx_ctl_packets",
 	"mac_tx_total_collisions",
 	"mac_tx_one_collision",
-	"mac_tx_multi_collison",
+	"mac_tx_multi_collision",
 	"mac_tx_max_collision_fail",
 	"mac_tx_max_deferal_fail",
 	"mac_tx_fifo_err",
